app/styles/components/_rui-avatar.sass

```diff
@@ -1,14 +1,28 @@
 .rui-avatar
-  width: 2.2857142857rem // ~32px at base 14
-  height: auto
+  background: url($default-avatar-small) top left no-repeat
+  background-size: 100% 100%
```

Comment:
`cover` was giving me a 1px artifact on the width.  True, on the 100%.   I'll remember that for next time ;)